Source/gg2/Objects/Alternative Objects/2014 Haxxy and Raybanns Rewards/Tombstone.events/Create.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<event category="CREATE" id="0">
+  <actions>
+    <action id="603" library="1">
+      <!--action name: Code-->
+      <kind>CODE</kind>
+      <allowRelative>false</allowRelative>
+      <question>false</question>
+      <canApplyTo>true</canApplyTo>
+      <actionType>CODE</actionType>
+      <functionName/>
+      <relative>false</relative>
+      <not>false</not>
+      <appliesTo>.self</appliesTo>
+      <arguments>
+        <argument kind="STRING">alarm[0] = 300/global.delta_factor;
+image_speed = 1/(3 * global.ticks_per_virtual);
+ownerClass = CLASS_QUOTE; // dummy class before being set
```

Comment:
IMO, the ownerClass doesn't really gain us much - we might as well just set the right image immediately in the Create event. I know GML doesn't have a good story for passing parameters into events, but I find it's acceptable practice to misuse global variables for parameter passing. We used the "param" prefix for that in other places, e.g.
`global.paramOwnerClass = victim.class;
instance_create(victim.object.x, victim.object.y, Tombstone);
profit();`